### DISCUSSION TITLE
Chaincode invocation always submitted to only one peer

### DISCUSSION THREAD

**Discussion by bh4rtp** [hyperledger/fabric-sdk-node#680] [Q&A] [answered]
The 2.2.16 sdk discovery feature and collections_config.json are used to build contract connection. The collections_config.json defines the requiredPeerCount=2. While every time the chaincode is invoked, only one CCAAS will receive the invocatoin.

Does node sdk support multiple peers endorsement?

**Comment by bh4rtp** [comment]
collections_config.json:
```
[
  {
    "name": "MyCollection",
    "policy": "OR('Org1.member')",
    "requiredPeerCount": 2,
    "maxPeerCount": 3,
    "blockToLive": 0,
    "memberOnlyRead": false,
    "memberOnlyWrite": false,
    "endorsementPolicy": {
      "signaturePolicy": "OR('Org1.member')"
    }
  }
]

````

**Comment by bestbeforetoday** [comment, marked as answer]
The peers (or rather organizations) used for endorsement are determined by the endorsement policy. The collection configuration options you are referring to define how private data is distributed between peers following endorsement. See the [Fabric documentation on collection definition](https://hyperledger-fabric.readthedocs.io/en/release-2.5/private_data_tutorial.html#pd-build-json).

**Comment by bh4rtp** [comment]
@bestbeforetoday So how can I set multiple endorsing peers for Org1? I configure them in the connection.json but does not work.
```
"channels": {
    "mychannel": {
      "orderers": [
        "orderer0.example.com",
        "orderer1.example.com",
        "orderer2.example.com"
      ],
      "peers": {
        "peer0.org1.example.com": {
          "endorsingPeer": true,
          "chaincodeQuery": true,
          "ledgerQuery": true,
          "eventSource": true
        },
        "peer1.org1.example.com": {
          "endorsingPeer": true,
          "chaincodeQuery": true,
          "ledgerQuery": true,
          "eventSource": true
        }
      }
    }
  },
```
It does work when using java sdk. What shall I do differently using node sdk?
